packages/json/json_type.pony

```diff
@@ -23,6 +23,13 @@ class JsonArray
     """
     data = data'
 
+  fun apply(): this->Array[JsonType] =>
+	"""
+	A getter to access the array data relative to the receiver's reference
```

Comment:
I think we can do better than these docs strings. I'm not sure they will mean much to folks. I think we would be better of with examples of usage perhaps here and definitely in examples in `json.pony`

Reply:
Note to self (@sgebbie), based on the 2022-02-01 sync discussion and input from Sean:

- provide examples of the cases of how `apply()` would be used in code, versus what would need to be done before introducing the `apply()` getter.
- include these examples in the JSON package documentation.